packages/grafana-data/rollup.config.mjs

```diff
@@ -4,7 +4,7 @@ import dts from 'rollup-plugin-dts';
 import esbuild from 'rollup-plugin-esbuild';
 import { externals } from 'rollup-plugin-node-externals';
 
-const pkg = require('./package.json');
+import pkg from './package.json' assert { type: 'json' };
```

Comment:
possibly stupid question - why do we need this new assert syntax? 🤔

Reply:
Good question @ashharrison90 . The file is now an es module and to use require we'd need to do something like:

```js
import { createRequire } from 'module';
const require = createRequire(import.meta.url);
const pkg = require('./package.json');
```

This felt a bit longwinded to me so I opted for the import assertion syntax. Admittedly it is still considered [experimental by nodejs](https://nodejs.org/dist/latest-v18.x/docs/api/esm.html#json-modules) so happy to change it if we're concerned it's gonna break at some point.